Jenkinsfile

```diff
@@ -54,6 +54,8 @@ pipeline {
 		sh "sudo sed -i -e 's/quay/docker/' /etc/containers/registries.conf"
 		sh "sudo curl -L https://github.com/opencontainers/runc/releases/download/$RUNC_VERSION/runc.amd64 -o /usr/bin/runc"
 		sh "sudo chmod +x /usr/bin/runc"
+		sh '''sudo curl -L https://dl.k8s.io/release/v${K8S_VERSION}/bin/linux/amd64/kubectl -o /usr/bin/kubectl'''
```

Comment:
why ''' is used here?

Reply:
I put first just "blah blah", but there was an error.
The error message said solution is to use ''' or """...
I don't know why it happens..

Reply:
Can you show the error?

Reply:
I don't understand why it does not reproduce the problem now.
But, unless I am insane, there would be no reason why I put ugly ''' out of the blue.

Ah, this Jenkinsfile is so tricky and stressful..